### PR TITLE
Make plots

### PR DESCRIPTION
**Pull request by kambysese**
@Eric89GXL maybe go with this for now for generating head movement plot. WDYT?

### CODE REVIEW

**Comment by Eric89GXL** [comment]
That looks like a good approach, except that it only plots the first run's head movement. I think it's fine to plot them separately or together, up to you.

**Comment by kambysese** [comment]
Oh hell! @Eric89GXL I probably should not have rebased before the recent commit? I only want to merge `d5bfc03`. How do I clean this up?

File: examples/funloc/analysis_fun.py [hunk 1]
```diff
@@ -123,6 +123,7 @@
     gen_ssp=True,       # Generate SSP vectors
     apply_ssp=True,     # Apply SSP vectors and filtering
     plot_psd=True,      # Plot raw data power spectra
+    plot_hp=True,       # plot head movement from raw data
```
Comment (Eric89GXL) [inline examples/funloc/analysis_fun.py]: we shouldn't need both a `params.` and a boolean here. Probably just the boolean here would be good

Comment (kambysese) [inline examples/funloc/analysis_fun.py]: Sorry slipped back in after I cleaned up a botched commit history.

File: mnefun/_mnefun.py [hunk 2]
```diff
@@ -239,6 +240,8 @@ class Params(Frozen):
     sss_origin : array-like, shape (3,) | str
         Origin of internal and external multipolar moment space in meters.
         Default is center of sphere fit to digitized head points.
+    plot_hp : bool
+        Plot head movement traces from raw data
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: this docstring is out of date. It's not part of the constructor anyway so let's get rid of it

**Comment by kambysese** [comment]
I compared with current master and deleted outdated commits. Should be ready for merge.

**Comment by Eric89GXL** [comment]
Thanks @kambysese

**Pull request merged**